plugin/trino-kinesis/src/main/java/io/trino/plugin/kinesis/s3config/S3TableConfigClient.java

```diff
@@ -99,7 +101,7 @@ protected void startS3Updates()
         // TODO: if required make the update interval configurable
         if (this.bucketUrl.isPresent()) {
             ScheduledExecutorService scheduler = Executors.newSingleThreadScheduledExecutor();
-            this.updateTaskHandle = scheduler.scheduleAtFixedRate(this::updateTablesFromS3, 5, 600, TimeUnit.SECONDS);
+            this.updateTaskHandle = scheduler.scheduleAtFixedRate(this::updateTablesFromS3, 5_000, tableDescriptionRefreshInterval.toMillis(), TimeUnit.SECONDS);
```

Comment:
Any reason for increase in the initial delay ? Can we capture it as a new commit. I guess the timeunit should be MILLISECONDS

Reply:
@somayaj Gentle reminder

Reply:
sorry, missed this. done